scripts/generate_checksum.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHASUM_FILES="shasums-eksctl-anywhere"
+FILE_PREFIX="eksctl-anywhere"
+
+DARWIN_TAR="darwin-amd64.tar.gz"
+LINUX_TAR="linux-amd64.tar.gz"
+
+mkdir -p $SHASUM_FILES
+cd $SHASUM_FILES
+
+release_manifest=$(curl --silent 'https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml' -o release.yaml)
+
+latest_release_version=$(cat release.yaml | yq e '.spec.latestVersion')
+
+latest_release=$(cat release.yaml | yq e '.spec.releases[] | select(.version == "'$latest_release_version'")' >> latest_release.yaml)
+
+linux_sha256=$(cat latest_release.yaml | yq e '.eksABinary.linux.sha256')
```

Comment:
There must be a way to put this in a loop over [256, 512] since we're doing very similar operations.